controlplane/pkg/nsmd/dataplanes.go

```diff
@@ -89,6 +89,7 @@ func dataplaneMonitor(model model.Model, dataplaneName string) {
 		// TODO: this is not good -- direct model changes
 		dataplane.RemoteMechanisms = updates.RemoteMechanisms
 		dataplane.LocalMechanisms = updates.LocalMechanisms
+		dataplane.RemoteConfigured = true
```

Comment:
Hmm what `RemoteConfigured` means here?

Reply:
Maybe should change it to RemoteMechanismChnaged, and clear the flag in NSM?

Reply:
We need that flag only to wait while nsmd will receive package with dataplane mechanisms parameters first time. So we don't need to clear the flag.
For example to fix situation, when we are trying to heal state with new registered dataplane, but we haven't received package with information about mechanisms yet.  Nsm will emit updateMechanism and return error because it cannot select mechanism.

Reply:
So RemoteConfigured means that we received at least one package with mechanisms and nsm can register connection

Reply:
OK, but those mechanisms can be "local" only, right?

Reply:
Yes

Reply:
I will rename it to MechanismsConfigured.